packages/components/cypress.config.js

```diff
@@ -9,6 +9,7 @@ module.exports = defineConfig({
     viewportHeight: 576,
   },
   includeShadowDom: true,
+  chromeWebSecurity: false,
```

Comment:
Can you please explain why it is necessary. Here is enough.

Reply:
For some reason I had CORS issues when testing locally. I might be able to remove it again, will retest.